bindings/c/src/models.rs

```diff
@@ -0,0 +1,82 @@
+use iota::client::Transfer;
+use iota::bundle;
+use iota::crypto::Kerl;
+use iota::signing::{IotaSeed, Seed};
+
+pub struct CSeed(pub(crate) IotaSeed<Kerl>);
+
+#[no_mangle]
+pub extern "C" fn iota_seed_new() -> *mut CSeed {
+    Box::into_raw(Box::new(CSeed(IotaSeed::new())))
+}
+
+#[no_mangle]
+pub extern "C" fn iota_seed_free(ptr: *mut CSeed) {
+    if ptr.is_null() {
+        return;
+    }
+    unsafe {
+        Box::from_raw(ptr);
```

Comment:
Shouldn't we use [`zeroize`](https://docs.rs/zeroize/1.1.0/zeroize/trait.Zeroize.html#tymethod.zeroize) to zero the memory before freeing the pointer?

Reply:
Thanks for the advise! I think we can even do it ourselves since it just one volatile and one atomic operation.